runtime/port/unix/j9sysinfo.c

```diff
@@ -1053,8 +1053,9 @@ getS390Description(struct J9PortLibrary *portLibrary, J9ProcessorDesc *desc)
 		if (J9_ARE_ALL_BITS_SET(auxvFeatures, J9PORT_HWCAP_S390_VXRS))
 #endif
 		{
+#if !defined(J9ZTPF)
```

Comment:
Would it be cleaner to have `getS390zOS_supportsVectorExtensionFacility()` return false for zTPF?

Reply:
@DanHeidinga Should I piggyback off of the existing zOS functions or should I create z/TPF versions... (e.g., getS390TPF_supportVectorExtensionFacility)?  The only reason I ask is those function are guarded with z/OS with some structure defintions.

Whatever you prefer.  tia..

Reply:
I'm not sure it makes sense to add a new function just to return false.  My suggestion for using the zOS functions was they help document the reason why (no support for vectors) this is being disabled.

Reply:
@DanHeidinga Actually we will try to fit into the Linux version using query_auxv.  Maybe hardcoding the flags as we don't have query_auxv on TPF.  We may implement one as well so that we completely are transparent with Linux configuration.  If that you are okay with that.

Reply:
That makes sense.

Reply:
This version sets the auxvFeatures variable as if it were set by an query_auxv call.  This allowed removing the z/tpF guard macros and making use of the J9_ARE_ALL_BITS_SET as if we were LINUX s390.